### PR TITLE
Add description on handling images

### PR DESCRIPTION
**Pull request by m-linner-ericsson**
### Applicable Issues
Action item from TC

### Description of the Change
Documenting how images should be handled within the Eiffel Community

### Alternate Designs
N/A

### Benefits
Guidelines for handling graphical images

### Possible Drawbacks
N/A

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Mattias Linner <mattias.linner@ericsson.com>

### CODE REVIEW

**Comment by m-linner-ericsson** [comment]
This could be too vague for a recommendation but let us see what you think

**Comment by e-backmark-ericsson** [comment]
In the logo section I believe you should include a link to working_with_community-logos.md

**Comment by m-linner-ericsson** [comment]
> 
> 
> In the logo section I believe you should include a link to working_with_community-logos.md

Fixed in f8922dc

**Review by e-backmark-ericsson — APPROVED** [review]

File: howtos/how-to-handle-graphical-images.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# How to handle graphical images
+
+This document describes how we handle logos and other graphical images.
+
+## General Recommendations
+When creating an graphical image, for use within the Eiffel community, add the original file to the repository along with any needed format. To help future updates, place a README in the directory explaining any procedures or thought behind the creative process.
```
Comment (magnusbaeck) [inline howtos/how-to-handle-graphical-images.md:6]: When creating **a** graphical image

File: howtos/how-to-handle-graphical-images.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<img alt="eiffel-intelligence-logo" src="https://raw.githubusercontent.com/eiffel-community/eiffel-intelligence/master/images/eiffel-intelligence-logo.png" width="350"/>
+<img alt="eiffel-playground" src="https://raw.githubusercontent.com/eiffel-community/eiffel-playground/master/images/logo.png" width="350"/>
+<img alt="eiffel-jenkins-plugin-logo" src="https://raw.githubusercontent.com/eiffel-community/eiffel-jenkins-plugin/master/images/logo.png" width="350"/>
+<img alt="eiffel-sepia-logo" src="https://raw.githubusercontent.com/eiffel-community/eiffel-sepia/master/images/logo.png" width="350"/>
+<img alt="eiffel-protocol-logo" src="https://raw.githubusercontent.com/eiffel-community/eiffel/master/images/eiffel-protocol-logo.png" width="350"/>
+
+## Graphical Images
+
+### SVG images
+Both the Eiffel protocol repository and Sepia have used [Inkscape](https://inkscape.org/). Inkscape saves its images in a variant of SVG that most browsers can render directly. When saving an image you get the choice to save it as "Inkscape SVG" or "Plain SVG". Always add the image as a Inkscape SVG to the repository.
```
Comment (magnusbaeck) [inline howtos/how-to-handle-graphical-images.md:34]: Always add the image as **an** Inkscape

**Review by magnusbaeck — CHANGES_REQUESTED** [review]
Two tiny grammatical nits that we fix as well fix, otherwise LGTM.

File: howtos/how-to-handle-graphical-images.md (hunk 1 above)
Comment (m-linner-ericsson) [reply howtos/how-to-handle-graphical-images.md]: Fixed in 7d3d01b

File: howtos/how-to-handle-graphical-images.md (hunk 2 above)
Comment (m-linner-ericsson) [reply howtos/how-to-handle-graphical-images.md]: Fixed in 7d3d01b

**Review by magnusbaeck — APPROVED** [review]

**Review by t-persson — APPROVED** [review]

**Pull request merged**